optimum/exporters/openvino/__main__.py

```diff
@@ -394,7 +394,7 @@ class StoreAttr(object):
             model = _OpenClipForZeroShotImageClassification.from_pretrained(model_name_or_path, cache_dir=cache_dir)
         else:
             model = TasksManager.get_model_from_task(
-                task,
+                "text-generation" if trust_remote_code and task == "image-text-to-text" else task,
```

Comment:
temporary fix : we currently load "image-text-to-text" models with `AutoModelForCausalLM` and not `AutoModelForImageTextToText`, this seems to be needed for remote code models like [phi3_v](https://huggingface.co/microsoft/Phi-3.5-vision-instruct/blob/main/config.json#L9) internvl2, minicpmv, internvl2, nanollava, maira2

WIll open a PR to load remote code models using `config.auto_map` instead (for other models we can directly use model.architectures)